frb_codegen/src/config.rs

```diff
@@ -482,3 +548,113 @@ impl Opts {
         .with_extension("freezed.dart")
     }
 }
+
+#[cfg(test)]
+mod test {
```

Comment:
Well, sorry but this may not be unit test... Unit test looks like:

```
    fn test_coutput_with_single_input_block_api() {
        let output = get_refined_c_output(vec!["api_1.rs"], vec!["./pathA/c_output.h"]);
        assert_eq!(output, vec!["the_value_that_you_expect_here"]);
  }
```

and write a few such tests, asserting scenarios like no extra output, have extra output, ...

Reply:
I've enumerated all cases in 6be714a6711f3dc08a3b6fcf1586bee077b3bfc3.

Reply:
Great! I will review soon